api/src/main/java/tc/oc/pgm/api/xml/Node.java

```diff
@@ -1,4 +1,4 @@
-package tc.oc.pgm.util.xml;
+package tc.oc.pgm.api.xml;
 
```

Comment:
I don't really think that the `tc.oc.pgm.api.xml` implementations belong in API, but a fair number of the API methods use them as parameters.

Reply:
Let's defer this to another phase, but yes, we should figure out how to remove this from `api`.